packages/scoped-elements/src/ScopedElementsMixin.js

```diff
@@ -95,7 +95,18 @@ const ScopedElementsMixinImplementation = superclass =>
         elementStyles,
       } = /** @type {typeof ScopedElementsHost} */ (this.constructor);
 
-      if (!this.registry) {
+      // @ts-ignore
+      const createRegistry =
```

Comment:
I would rename the variable to something like `shouldCreateRegistry` or `registryNotCreated` to reflect an intention or state instead of an action.

Reply:
Done